services/device-registry/src/test/resources/device-identities.json

```diff
@@ -29,9 +29,9 @@
       },
       {
         "device-id": "gw-1",
-        "comment": "this device is authorized to publish data on behalf of device 4712",
         "data": {
-          "enabled": true
+          "enabled": true,
+          "comment": "this device is authorized to publish data on behalf of device 4712"
```

Comment:
what't the purpose of this?

Reply:
The device registry accepts the comment inside of `data` and ignores it being outside. I found itw hen I configured the device registry (started from the IDE) to use this file. It reads it and then stores it without the comment. Not a big issue tbh, but irritating.